src/main/kotlin/org/rust/ide/refactoring/extractFunction/ExtractFunctionParameterTablePanel.kt

```diff
@@ -17,6 +23,57 @@ class ParameterDataHolder(val parameter: Parameter, val onChange: () -> Unit) :
     }
 }
 
+class ChooseColumn : ColumnInfo<ParameterDataHolder, JBCheckBox>(null) {
```

Comment:
Maybe use `Boolean` instead of `JBCheckBox`? Please check `com.intellij.util.ui.ColumnInfo` usages and find a proper way.